### ISSUE TITLE
Allow comparing schema dumps

### ISSUE THREAD

**Issue by yashsriv** [djrobstep/migra#27]
Hi, is there any way I could compare two schema dumps?

As far as I can see in the code, the library you have used can only work with SQLAlchemy Connections. Would there be some way to create a mock connection with data from the schema dump sql files.

**Comment by thugdoubt** [comment]
@yashsriv there's an app for that: https://www.apgdiff.com/

**Comment by djrobstep** [comment]
Yes migra works with actual connections, not dump files. But you can just create two temporary databases, load the files, and compare them once loaded.

[In the docs] (http://migra.djrobstep.com/dev_usage/) there's some code that shows you how to spin up temporary databases just for this purpose.

**Comment by jflambert** [comment]
@thugdoubt Sorry but apgdiff is broken for PostgreSQL 10 and up
